frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -665,11 +665,12 @@ class DataApi {
    *
    * @example
    * const position = [123, 123, 123];
-   * const segmentId = await api.data.getDataValue("segmentation", position);
+   * const segmentationLayerName = "segmentation";
```

Comment:
@philippotto I slightly changed the example as I misunderstood it by only giving it a quick look. In my opinion, the newer version makes it more clear, that the name of the segmentation layer is not always "segmentation".